app/jobs/activity_tracking_job.rb

```diff
@@ -0,0 +1,23 @@
+class ActivityTrackingJob < ApplicationJob
+  queue_as :dradis_project
+
+  def perform(action:, trackable_id:, trackable_type:, user:)
+    trackable = trackable_type.constantize.find(trackable_id) rescue nil
```

Comment:
@dormi looks like an anti-pattern to rescue exceptions here. Might be better to use another method instead to check for existence?